martian/src/mro.rs

```diff
@@ -25,11 +25,11 @@ use std::string::ToString;
 /// is disallowed.
 pub const MARTIAN_TOKENS: &[&str] = &[
     "in", "out", "stage", "volatile", "strict", "true", "split", "filetype", "src", "py", "comp",
-    "retain", "mro", "using", "int", "float", "string", "map", "bool", "path", "__null__",
+    "retain", "mro", "using", "int", "float", "string", "map", "bool", "path",
```

Comment:
I would keep `__null__` for now. It was needed as a workaround here with the older serde-json crate: https://github.com/martian-lang/martian-rust/blob/master/martian/src/stage.rs#L14-L22

Reply:
It's redundant with the prohibition on a double underscore prefix.